Dockerfile

```diff
@@ -2,8 +2,8 @@
 # Not to be used in production environment (yet).
 FROM python:3.6
 
-WORKDIR /usr/src/app
-COPY . /usr/src/app
+WORKDIR /app
```

Comment:
Why does changing this matter?

Reply:
@nlMeminger For simplicity. The standard path I normally see used for containers is something like `/app` or `/data` in the root directory. This is only a cosmetic change.